src/game/coreObjects/cell/EmptyCell.js

```diff
@@ -28,7 +29,9 @@ class EmptyCell extends Cell {
 
   // The meter either goes towards 0 or 1 based on whether or not the piece is marked.
   idleUpdate() {
-    super.idleUpdate();
+    if (this.coreState.settingsController && this.coreState.settingsController.graphicsLevel == Setting.HIGH) {
```

Comment:
Don't use conditional updates at the subclass level, I may not get the position offset updates from clearing lines. Just handle it at the Cell superclass